qiskit_aer/primitives/estimator.py

```diff
@@ -100,7 +107,15 @@ def __init__(
         self._transpile_options = Options()
         if transpile_options is not None:
             self._transpile_options.update_options(**transpile_options)
-        self.approximation = approximation
+        if not approximation:
+            warn(
+                "Option approximation=False is deprecated as of qiskit-aer 0.13. "
+                "It will be removed no earlier than 3 months after the release date. "
+                "Instead, use BackendEstmator from qiskit.primitives.",
```

Comment:
I just happened to notice this PR - in looking over it I see that BackendEst***i***mator is missing the ***i*** in this warning - same in the one below. Pretty minor but I thought I'd point it out.